src/pages/cheoljinPark/Main/Feeds.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import Comments from './Comments';
+import './Feeds.scss';
+import { AiTwotoneHeart } from 'react-icons/ai';
+import { BiCommentDetail } from 'react-icons/bi';
+import { MdOutlineIosShare } from 'react-icons/md';
+import { BsThreeDots, BsBookmarkFill } from 'react-icons/bs';
+
+function Feeds() {
+  let [commentText, setCommentText] = useState('');
+  let [commentList, setCommentList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/cheoljinPark/mock.json')
+      .then(res => res.json())
+      .then(data => setCommentList(data));
+  }, []);
+
+  const commentChangeHandle = event => {
```

Comment:
```suggestion
  const getComment = event => {
```
인풋의 value를 가져오는 함수니까 getComment로 이름 수정해보면 좋을 것 같아요 (성훈님 함수명..)